pages/api/sheeter.js

```diff
@@ -73,11 +73,11 @@ const getFormData = async (sheets_client) => {
 const sendFormResponse = async (sheets_client, form_data) => {
   var params = {
     // The ID of the spreadsheet to update.
-    spreadsheetId: "1d4UQSXPPOcjFtu7RkJsh6DvhRKt11xVCW8WyTC8GeqE", // personal
+    spreadsheetId: process.env["SURVEY_SHEET_ID"],
```

Comment:
Good idea to switch to `process.env` 👍

Reply:
Thx for your review @OlaShabalina ! 🙇🏽 

I had to pull out `SURVEY_SHEET_ID` env var from `.env`, and let Vercel manage that for us instead. Reason for that was because I needed to provide a different sheet id for the [production](http://seechange.codeforaustralia.org/) deployment, and the preview deployment. I don't want test submissions from the preview deployment to end up in the same spreadsheet that will contain all of council staffs' submissions!

Reply:
I see, no worries! 👍😄